src/js/outreach/index.js

```diff
@@ -220,6 +219,17 @@ function startOutreachApp() {
   Backbone.history.start({ pushState: true });
 }
 
+let token;
+
+Radio.reply('auth', {
```

Comment:
Add `auth` Radio to `outreach` as now `js/auth` isn't in this js bundle at all, _but_ the `getToken` in the base fetch _will_ use this `Radio` for this particular bundle, but the `auth.js` `Radio` for the main bundle.